### PR TITLE
Update gulpfile to only process scss

### PR DESCRIPTION
**Pull request by joshblum**

### CODE REVIEW

File: orchestra/static/orchestra/common/css/lib/style.css [hunk 1]
```diff
@@ -248,27 +248,6 @@ ul.top-menu > li > .logout {
     padding-bottom: 10px;
 }
 
-/*LEFT NAVIGATION ICON*/
```
Comment (joshblum) [inline orchestra/static/orchestra/common/css/lib/style.css:251]: note: these image files are missing and fail the whitenoise build. the class is unused, @derek-schultz says it's ok to remove

File: gulpfile.js [hunk 2]
```diff
@@ -19,13 +25,10 @@ var files = {
...
     'orchestra/static/orchestra/common/css/orchestra.scss',
-    'orchestra/static/orchestra/common/css/registration.scss',
-    'orchestra/static/orchestra/dashboard/css/dashboard.scss',
-    'orchestra/static/orchestra/task/css/task.scss',
   ],
   all_scss: [],
-  scripts: [
-    '!' + buildDestination + '/**',
+  jslint: [
+    './gulpfile.js', // Lint ourselves!
```
Comment (marcua) [inline gulpfile.js:31]: :)

**Comment by marcua** [comment]
lgtm, with one concern---a bunch of files were deleted and added, rather than renamed---can we just rename?

**Comment by joshblum** [comment]
handled offline. we good

**Pull request merged**